src/bus.js

```diff
@@ -0,0 +1,24 @@
+export default {
+    events: {},
+
+    on(name, callback) {
+        if (! this.events.hasOwnProperty(name)) {
+            this.events[name] = []
+        }
+
+        this.events[name].push(callback)
+    },
+
+    emit(name, data = {}) {
+        if (this.events.hasOwnProperty(name)) {
+            this.events[name].forEach(callback => {
+                callback(data)
+            })
+        }
+
+        window.dispatchEvent(new CustomEvent(`spruce:${name}`, {
```

Comment:
Um IE11 won't like this

I should really get round to creating that `browser-dispatch` module 🙈

Reply:
Yeah haha, I've got another issue (#18) to add polyfills for IE11 support.

Reply:
IE11 will want different methods and there's no CustomEvent constructor I think, maybe worth leaving a "deal with this for IE11" comment

Reply:
Yeah good shout